install-purescript/index.js

```diff
@@ -26,9 +23,8 @@ function addId(obj, id) {
 
 const defaultCacheRootDir = envPaths('purescript-npm-installer').cache;
 const CACHE_KEY = 'install-purescript:binary';
-const MAX_READ_SIZE = 30 * 1024 * 1024;
 const defaultBinName = `purs${process.platform === 'win32' ? '.exe' : ''}`;
-const cacheIdSuffix = `-${process.platform}-${arch()}`;
+const cacheIdSuffix = `-${process.platform}-${process.arch}`;
```

Comment:
Not 100% sure what `cacache` does with this but if it doesn't match the logic we're using in `download-purescript` then might that result in cache misses?

Reply:
We'll have a cache miss if the user changes architecture, which I think it's uncommon enough :)